packages/jsii-rosetta/test/rosetta.test.ts

```diff
@@ -10,20 +17,29 @@ const SAMPLE_CODE: TypeScriptSnippet = {
   location: testSnippetLocation('sample'),
 };
 
-test('Rosetta object can do live translation', () => {
-  // GIVEN
-  const rosetta = new Rosetta({
-    liveConversion: true,
-    targetLanguages: [TargetLanguage.PYTHON],
+describe('Rosetta object can do live translation', () => {
+  let rosetta: Rosetta;
+  let translated: Translation | undefined;
+  beforeEach(() => {
```

Comment:
Wondering about using `beforeEach` vs `beforeAll`... It seems like `beforeAll` would work here since the tests are not modifying `rosetta` or `translated` (yay it saves 1 translation) however it might make more sense to default to `beforeEach` as it is safer.

My question is, did you mean `beforeAll`? Or do you like to default to `beforeEach` whenever possible?

Reply:
I did have `beforeAll` at first, but I felt `beforeEach` was safer.

I would say the rule would be:

* Default to using `forEach`, it is most likely going to save you trouble if it so happens that things are accidentally mutable, or become mutable after a while (even though you didn't intend them to be at the start).
* If the performance cost of `beforeEach` is horrible, fall back to `beforeAll` (and make *damn* sure your fixture is truly immutable)